[2025-02-08T15:27:19.776+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-08T15:27:19.827+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: directory_and_file_verification_dag.check_directory manual__2025-02-08T15:27:16.399898+00:00 [queued]>
[2025-02-08T15:27:19.852+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: directory_and_file_verification_dag.check_directory manual__2025-02-08T15:27:16.399898+00:00 [queued]>
[2025-02-08T15:27:19.854+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-02-08T15:27:19.899+0000] {taskinstance.py:2889} INFO - Executing <Task(BranchPythonOperator): check_directory> on 2025-02-08 15:27:16.399898+00:00
[2025-02-08T15:27:19.918+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=3163) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-08T15:27:19.923+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'directory_and_file_verification_dag', 'check_directory', 'manual__2025-02-08T15:27:16.399898+00:00', '--job-id', '1258', '--raw', '--subdir', 'DAGS_FOLDER/my_directory_and_file_checker.py', '--cfg-path', '/tmp/tmpd68srlgl']
[2025-02-08T15:27:19.926+0000] {standard_task_runner.py:72} INFO - Started process 3172 to run task
[2025-02-08T15:27:19.927+0000] {standard_task_runner.py:105} INFO - Job 1258: Subtask check_directory
[2025-02-08T15:27:20.054+0000] {task_command.py:467} INFO - Running <TaskInstance: directory_and_file_verification_dag.check_directory manual__2025-02-08T15:27:16.399898+00:00 [running]> on host 003b7fbea265
[2025-02-08T15:27:20.295+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='directory_and_file_verification_dag' AIRFLOW_CTX_TASK_ID='check_directory' AIRFLOW_CTX_EXECUTION_DATE='2025-02-08T15:27:16.399898+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-08T15:27:16.399898+00:00'
[2025-02-08T15:27:20.300+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-02-08T15:27:20.301+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-02-08T15:27:20.302+0000] {logging_mixin.py:190} INFO - Current task name:check_directory state:running start_date:2025-02-08 15:27:19.829375+00:00
[2025-02-08T15:27:20.304+0000] {logging_mixin.py:190} INFO - Dag name:directory_and_file_verification_dag and current dag run status:running
[2025-02-08T15:27:20.305+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-08T15:27:20.307+0000] {baseoperator.py:421} WARNING - BranchPythonOperator.execute cannot be called outside TaskInstance!
[2025-02-08T15:27:20.310+0000] {python.py:240} INFO - Done. Returned value was: check_file_in_directory
[2025-02-08T15:27:20.311+0000] {branch.py:38} INFO - Branch into check_file_in_directory
[2025-02-08T15:27:20.312+0000] {skipmixin.py:230} INFO - Following branch ('check_file_in_directory',)
[2025-02-08T15:27:21.056+0000] {skipmixin.py:278} INFO - Skipping tasks []
[2025-02-08T15:27:21.133+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-08T15:27:21.135+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=directory_and_file_verification_dag, task_id=check_directory, run_id=manual__2025-02-08T15:27:16.399898+00:00, execution_date=20250208T152716, start_date=20250208T152719, end_date=20250208T152721
[2025-02-08T15:27:21.163+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-02-08T15:27:21.164+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-02-08T15:27:21.165+0000] {logging_mixin.py:190} INFO - Dag name:directory_and_file_verification_dag queued_at:2025-02-08 15:27:16.418545+00:00
[2025-02-08T15:27:21.167+0000] {logging_mixin.py:190} INFO - Task hostname:003b7fbea265 operator:BranchPythonOperator
[2025-02-08T15:27:21.234+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-08T15:27:21.312+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-08T15:27:21.317+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
